MONITORING.md

```diff
@@ -0,0 +1,28 @@
+The `kudo-spark-operator` is able to seamlessly integrated with Prometheus, which is installed within [Prometheus operator](https://github.com/coreos/prometheus-operator).
+Integration with other Prometheus distributions, like kube-prometheus, wasn't tested. 
+
+The `prometheus-operator` uses crafted services discovery approach, introducing `ServiceMonitor` kind. 
+But the `kudo-spark-operator` take its configuration burden on itself.
+
+#### How to get `kudo-spark-operator` and app's metrics available for scrape by Prometheus:
+1) Ensure `prometheus-operator` is installed on your Kubernetes cluster.
+1) Install the `kudo-spark-operator` with `-p enableMetrics=true` parameter. (But it's true by default, so typically no need to specify it explicitly)
+1) Composing your Spark Application yaml:
+   - use Spark image with JMXPrometheus exporter jar on the board i.e. `gcr.io/spark-operator/spark:v2.4.4-gcs-prometheus` 
+   - enable driver/executors monitoring by adding the yaml piece into `spec` section:
+     ```yaml
+       monitoring:
+         exposeDriverMetrics: true
+         exposeExecutorMetrics: true
+         prometheus:
+           jmxExporterJar: "/prometheus/jmx_prometheus_javaagent-0.11.0.jar"
+           port: 8090
+     ```  
+   - if you would like use other than 8090 port for metrics exporting, you must pass appropriate parameter during `kudo-spark-operator` installation `-p appMetricsPort=<desired_port>` 
+   - Mark your application by the label 
+     ```yaml
+       labels:
+         exposedMetrics: true
+     ```
+
+Full blown example you can find in [spark-application.yaml](specs/spark-application.yaml).
```

Comment:
```suggestion
Full configuration example is available in [specs/spark-application.yaml](specs/spark-application.yaml).
```

Reply:
Could you, pls, shine light on why the wording better?

Reply:
Sure, here are some points:
* original sentence lacks the subject (example of what?)
* word ordering issue (should be 'you can find the example')
* we avoid using pronouns ('you') in the service docs
* 'Full blown' doesn't suit the context here (see [Merriam-Webster](https://www.merriam-webster.com/dictionary/full-blown) definition and synonyms)
* `spark-application.yaml` -> `specs/spark-application.yaml` in the link name provides the location of the file in the project,  not just the link

Does it make sense?

Reply:
@akirillov, it's sweet :-) Thanks for the explanation. It makes sense. Just a couple of comments from my side:  
> original sentence lacks the subject (example of what?)

neat!

> we avoid using pronouns ('you') in the service docs

Good point. And If you have somewhere guide/list of best practices like that, which we should follow. I'd appreciate if you will share it with entire team.

> 'Full blown' doesn't suit the context here (see Merriam-Webster definition and synonyms)

I do not insist to use `Full blown' phrase here :-) just to debate - as per [Collins dictionary](https://www.collinsdictionary.com/dictionary/english/full-blown) it also has meaning "fully grown or developed;", which is suitable for this context, imo.

> spark-application.yaml -> specs/spark-application.yaml in the link name provides the location of the file in the project, not just the link

mb it makes sense for better transparency. Beside of that user always can see location in url or, after click, in github ui.

Reply:
you can check other services' docs created by the team for the inspiration regarding the style and format e.g. [Data Science Engine](https://docs.d2iq.com/mesosphere/dcos/services/data-science-engine/1.0.0/) or [DCOS Spark](https://docs.d2iq.com/mesosphere/dcos/services/spark/2.9.0-2.4.3/). I think Docs team has some guidelines but usually, we follow review suggestions.

RE: 'full blown', I don't think it's wrong it's just doesn't suit the context: 'fully grown configuration example' or 'fully developed configuration example' (developed not in the software sense). I'd expect 'full' or 'complete' here, but it's up to you.

Reply:
ok, thanks, make sense